doc/articles/getting-started-tutorial-1.md

```diff
@@ -149,7 +149,7 @@ To run the WebAssembly (Wasm) head, select **IIS Express** and press **Ctrl+F5**
     ```
 
     > [!IMPORTANT]
-    > Because we want to be able to respond to changes in our model we'll want to bring in the ObservableObject in the MvvmHelpers namespace, from the Refactored.MvvmHelpers NuGet package we installed earlier.
+    > Because we want to be able to respond to changes in our model we'll want to bring in the ObservableObject in the MvvmHelpers namespace, from the Refactored.MvvmHelpers NuGet package we installed earlier, in .
```

Comment:
```suggestion
    > Because we want to be able to respond to changes in our model we'll want to bring in the ObservableObject in the MvvmHelpers namespace, from the Refactored.MvvmHelpers NuGet package we installed earlier.
```